ivy/data_classes/array/statistical.py

```diff
@@ -14,7 +14,9 @@ def min(
         /,
         *,
         axis: Optional[Union[int, Sequence[int]]] = None,
-        keepdims: bool = False,
+        keepdims: Optional[bool] = False,
```

Comment:
actually the `Optional` typehint should only be used when the default value of the argument is `None`. Same applies to the container methods, ivy function and backend functions 😄

Reply:
@vedpatwardhan Done the required changes. One thing I want to ask if you see how i have handled the various datatypes using if else, is that the only option I have if I want to incorporate them? I cant use `inf` for `integer` and `unsigned` as it gives `EagerTensor cannot convert `inf` to int32` or like that. So I chose this to replace the `max` value.

Reply:
I think this could be simplified further as mentioned in the review comment, thanks!